Client/src/Unified/Services/Submitter/Service.cs

```diff
@@ -56,6 +56,8 @@ namespace ArmoniK.DevelopmentKit.Client.Unified.Services.Submitter;
 [MarkDownDoc]
 public class Service : AbstractClientService, ISubmitterService
 {
+  private const int MaxRetries = 5;
```

Comment:
this value should be a parameter to be able to modify it if needed by the extensions.csharp user